requirements.bazel.txt

```diff
@@ -3,16 +3,16 @@ coverage==4.5.4
 cython==0.29.21
 protobuf>=3.5.0.post1, < 4.0dev
 wheel==0.38.1
+google-auth==1.24.0
```

Comment:
It's not new, I removed them in this PR: https://github.com/grpc/grpc/pull/33384 because they're not compatible with  `opencensus-ext-stackdriver`, but we're actually using them in our tests:
https://github.com/grpc/grpc/blob/50b654894ad0d5b13a572707b659a7b3038f1b62/src/python/grpcio_tests/tests/interop/BUILD.bazel#L37

Not sure why that PR passed all tests.